### PR TITLE
[ENG-1650] Sloan querying attributes fix

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

Oh with the excitement of it all we ding-dang forgot to actually map the new sloan attributes to the PreprintView/Download templates, this adds them in and adds tests to make sure we never forget.

## Changes

- adds sloan fields to Preprint metrics model
- adds tests and travis changes.

## QA Notes

Devtest and Courtney test only.

## Documentation

🐞 fix, no new docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-1650

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ env:
     - LIBXML2_DEB="libxml2-dbg_2.9.1+dfsg1-3ubuntu4.9_amd64.deb"
     - POSTGRES_DEB="postgresql-9.6_9.6.3-1.pgdg12.4+1_amd64.deb"
     - ELASTICSEARCH_ARCHIVE="elasticsearch-2.4.5.tar.gz"
+    - ELASTICSEARCH_ARCHIVE6="elasticsearch-6.3.1.tar.gz"
```
Comment (brianjgeiger) [inline .travis.yml]: Super-minor: Shouldn't this be `ELASTICSEARCH6_ARCHIVE`?

Comment (Johnetordoff) [reply .travis.yml]: quite right

File: .travis.yml [hunk 2]
```diff
@@ -27,6 +27,8 @@ env:
     - LIBXML2_DEB="libxml2-dbg_2.9.1+dfsg1-3ubuntu4.9_amd64.deb"
     - POSTGRES_DEB="postgresql-9.6_9.6.3-1.pgdg12.4+1_amd64.deb"
     - ELASTICSEARCH_ARCHIVE="elasticsearch-2.4.5.tar.gz"
+    - ELASTICSEARCH6_ARCHIVE="elasticsearch-6.3.1.tar.gz"
+    - ELASTICSEARCH_HOST="127.0.0.1:9201"
```
Comment (mfraezz) [inline .travis.yml]: This appears unused, and were it used also should be named `ELASTICSEARCH6_HOST`, I believe.

File: .travis.yml [hunk 3]
```diff
@@ -96,6 +98,33 @@ before_install:
...
 
+    # elasticsearch6
+    - |
+
+      if [ ! -f "$ELASTICSEARCH6_ARCHIVE" ]; then
+        curl -SLO https://artifacts.elastic.co/downloads/elasticsearch/$ELASTICSEARCH6_ARCHIVE
+      fi
+
+      if [ ! -f "$ELASTICSEARCH6_ARCHIVE.sha1.txt" ]; then
+        curl -SLO https://artifacts.elastic.co/downloads/elasticsearch/ELASTICSEARCH6_ARCHIVE.sha1.txt
```
Comment (mfraezz) [inline .travis.yml]: Does this need a `$` before `ELASTICSEARCH6_ARCHIVE`?

File: api_tests/conftest.py [hunk 4]
```diff
@@ -14,3 +16,28 @@ def app():
 @pytest.fixture(autouse=True, scope='session')
 def app_init():
     init_app(routes=False, set_backends=False)
+
+
+@pytest.fixture(scope='function')
+def es_client():
```
Comment (mfraezz) [inline api_tests/conftest.py]: Minor, but for disambiguation this seems like the sort of thing that should maybe have a `6`

File: pytest.ini [hunk 5]
```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --ds=osf_tests.settings --tb=short --reuse-db --allow-hosts=127.0.0.1
+addopts = --ds=osf_tests.settings --tb=short --reuse-db --allow-hosts=127.0.0.1,192.168.168.167
```
Comment (mfraezz) [inline pytest.ini]: Why is this necessary, local testing?

Comment (Johnetordoff) [reply pytest.ini]: yes, it would be nice if we could just leave this in
